synonyms.tsv

```diff
@@ -1363,3 +1363,4 @@ CONSO00352	Mini-Mental_Status_Exam	ncit:C74982	EXACT
 CONSO00353	1-(5-acetylamino-5-carboxypentyl)-3-hydroxy-5-hydroxymethyl-pyridinium	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde derived Pyridinium compound	pmid:12784645	EXACT
 CONSO00353	Glyceraldehyde-derived Pyridinium compound	pmid:12784645	EXACT
+CONSO00363	N-(6-(2-amino-4-fluorophenylamino)-6-oxohexyl)-4-methylbenzamide	mesh:C556032	EXACT
```

Comment:
Add the other lexical variants mentioned in CT’s with different spacing and dashes